src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    MetadataInstanceForm,
+    type MetadataTemplateInstance,
+    UnsavedChangesModal,
+    type MetadataTemplate,
+} from '@box/metadata-editor';
+import React from 'react';
+
+export interface MetadataInstanceEditorProps {
+    isBoxAiSuggestionsEnabled: boolean;
+    isUnsavedChangesModalOpen: boolean;
+    template: MetadataTemplateInstance | MetadataTemplate;
+}
+
+const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
+    isBoxAiSuggestionsEnabled,
+    isUnsavedChangesModalOpen,
+    template,
+}) => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const handleSubmit = () => {}; // TODO in a future PR
```

Comment:
Instead of adding the eslint flag can you just add //TODO in a future PR as a comment like this

const handleSubmit = () => { // TODO in a future PR}

Reply:
changed that, thank you